cluster-autoscaler/core/utils/test_utils.go

```diff
@@ -32,7 +32,7 @@ func GetStringFromChan(c chan string) string {
 	select {
 	case val := <-c:
 		return val
-	case <-time.After(100 * time.Millisecond):
+	case <-time.After(2 * time.Second):
```

Comment:
nit: Why bump this? Won't it increase the time it takes to run tests?

Reply:
I set NodeDeleteDelayAfterTaint to 1second in newWrapperForTesting and some tests time out with the previous wait time. You are correct, it will increase the time needed for tests so I changed it to 0 seconds.